src/Metadata/AutoIncrementMethodType.cs

```diff
@@ -18,5 +19,11 @@ public enum AutoIncrementMethodType
         /// KORM generate primary key for entity.
         /// </summary>
         Custom = 1,
+
+        /// <summary>
+        /// Sql server generates the primary key and KORM fills it into the entity.
+        /// </summary>
+        /// <remarks>When calling <see cref="IDbSet.BulkInsert"/> keys are not fill back to the entities.</remarks>
```

Comment:
I would put this info directly into `summary` - it is important so let's make it visible. And that `<see cref=... />` generates warning on my computer. `cref` should be `IDbSet{T}.BulkInsert()`.